CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 * Optimize transactions queries by changing indexes order
 * Update cozy-ui to 51.4.0
 * Update cozy-harvest-lib to 6.4.0
+* Update cozy-client and cozy-pouch-link to 24.1.0
```

Comment:
Je pense qu'il est important d'expliquer ce qui t'intéresse. Genre fix queryAll bug and destroy on Pouch

Reply:
et j'aurais squasher le commit genre avec le suivant ?